src/main/scala/no/ndla/articleapi/service/ConverterService.scala

```diff
@@ -153,6 +153,20 @@ trait ConverterService {
       (toKeep ++ updated).filterNot(_.tags.isEmpty)
     }
 
+    def updateExistingTags(existingTags: Seq[ArticleTag], updatedTags: Seq[ArticleTag]): Seq[ArticleTag] = {
+      val tagsToKeep = updatedTags.filter(tag => existingTags.map(_.language).contains(tag.language))
+      val restTags = existingTags.filterNot(tag => tagsToKeep.map(_.language).contains(tag.language))
+      tagsToKeep ++ restTags
+    }
+
+    def updateExistingArticleMetaDescription(
+        existingMetaDesc: Seq[ArticleMetaDescription],
+        updatedMetaDesc: Seq[ArticleMetaDescription]): Seq[ArticleMetaDescription] = {
+      val metaDescToKeep = updatedMetaDesc.filter(tag => existingMetaDesc.map(_.language).contains(tag.language))
+      val restMetaDesc = existingMetaDesc.filterNot(tag => metaDescToKeep.map(_.language).contains(tag.language))
+      metaDescToKeep ++ restMetaDesc
+    }
```

Comment:
Syns disse (`updateExistingTags` og `updateExistingArticleMetaDescription`) var litt vanskelige å lese, men tror vi kan komme langt med litt bedre navn.

Føler selve funksjonsnavnet sier lite om at funksjonen kun oppdaterer språk som allerede finnes (Måtte kikke både 2 og 3 ganger for å skjønne hvorfor vi ikke bare kunne bruke `mergeLanguageFields` funksjonen).

`metaDescToKeep` er vel de nye metabeskrivelsene, så kanskje noe som `newMetaDescriptions` er bedre?
Syns egentlig `metaDescToKeep` er et bedre navn på `restMetaDesc` variabelen siden den filtrerer de vi faktisk skal spare på.

Reply:
Skjønner hva du mener, prøvde å kommunisere `funksjonen kun oppdaterer språk som allerede finnes` ved å bruke `updateExisting` som funksjonsnavnet, men kanskje det kom ikke nok frem. Har du noe forslag hva funksjonene kan kalles?